src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift

```diff
@@ -822,9 +822,12 @@ class HomeTableViewController: UITableViewController, NavigationBarOpacityDelega
 			return
 		}
 
+		let currentAppConfig = appConfigurationProvider.currentAppConfig.value
+		let numberOfDays = currentAppConfig.hasRiskCalculationParameters ? Int(currentAppConfig.riskCalculationParameters.maxEncounterAgeInDays) : 10
```

Comment:
#4325 seems to use a different approach here and defaults to 14, maybe the property from that PR can be used here as well?

Reply:
It's not available on the release branch yet - I can try to cherry pick it.